blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/service/ShardStateIntegrationTest.java

```diff
@@ -414,21 +423,21 @@ public void testSlotStateConvergence() throws InterruptedException {
         List<ShardStateWorker> allWorkers = new ArrayList<ShardStateWorker>(6);
 
         // Ingestor 1
-        ScheduleContext ctxIngestor1 = new ScheduleContext(time, shards);
+        ScheduleContext ctxIngestor1 = new ScheduleContext(time, shards, new DefaultClockImpl());
         ShardStatePuller pullerIngestor1 = new ShardStatePuller(shards, ctxIngestor1.getShardStateManager(), this.io);
         ShardStatePusher pusherIngestor1 = new ShardStatePusher(shards, ctxIngestor1.getShardStateManager(), this.io);
         allWorkers.add(pullerIngestor1);
         allWorkers.add(pusherIngestor1);
 
         // Ingestor 2
-        ScheduleContext ctxIngestor2 = new ScheduleContext(time, shards);
+        ScheduleContext ctxIngestor2 = new ScheduleContext(time, shards, new DefaultClockImpl());
```

Comment:
Dont you want to change all of these to use the constructor that takes 2 params?